### DISCUSSION TITLE
Serialization of 'Closure' is not allowed

### DISCUSSION THREAD

**Discussion by alancwoo** [wp-graphql/wp-graphql-smart-cache#283] [Q&A]
Recently, with:

1.  WPGraphQL Smart Cache 1.3.2
2. WPGraphQL 1.26.0
3. WPGraphQL for ACF 2.2.0

I am getting a `NO SCHEMA AVAILABLE` error in the IDE and also seeing errors in the console: `Serialization of 'Closure' is not allowed`

It seemed to work a few weeks ago, I also tried downgrading several versions to `WPGraphQL 1.24.0` and `WPGraphQL Smart Cache 1.3.01 to no avail. Oddly, it worked a few weeks ago and no new plugins were introduced since. Also tried using a default theme with no custom types or taxonomies.

This occurs when the Object cache is enabled.

**Comment by catative** [comment]
Hellow, @alancwoo. I think this might be helpful for you: https://github.com/wp-graphql/wp-graphql-smart-cache/issues/298
